aamt_parse2.pm

```diff
@@ -873,7 +1136,7 @@ sub deployToSite
             $nBar = $barWidth;
         }
         
-        ilog_print(1,"[" . "=" x $nBar . " " x ($barWidth - $nBar) . "]\r");
+#        ilog_print(1,"[" . "=" x $nBar . " " x ($barWidth - $nBar) . "]\r");
```

Comment:
You need to add a cmd line flag for running in unattended/non-interactive mode. If this flag is true, then you can skip this, if it is false, then keep the current code.